packages/appsync-modelgen-plugin/src/preset.ts

```diff
@@ -4,6 +4,7 @@ import { join } from 'path';
 import { JAVA_SCALAR_MAP, SWIFT_SCALAR_MAP, TYPESCRIPT_SCALAR_MAP, DART_SCALAR_MAP, METADATA_SCALAR_MAP } from './scalars';
 import { LOADER_CLASS_NAME, GENERATED_PACKAGE_NAME } from './configs/java-config';
 import { graphqlName, toUpper } from 'graphql-transformer-common';
+import { ObjectTypeDefinitionNode, StringValueNode } from "graphql/language/ast";
```

Comment:
Forgot to remove these in the refactor (https://github.com/aws-amplify/amplify-codegen/pull/725). I believe they are no longer used.
```suggestion
```